src/modules/addTask.js

```diff
@@ -0,0 +1,28 @@
+import itemsArray from './itemsArray.js';
+import renderToDoItems from './renderTasks.js';
+
+const addElBtn = document.querySelector('.to-do-addTask');
+const input = document.querySelector('input');
+const todoinputEl = document.querySelector('.to-do-input');
+
+const addTask = () => {
+  addElBtn.addEventListener('click', () => {
+    const newTask = todoinputEl.value;
+    if (newTask) {
+      const newTodo = {
+        description: newTask,
+        completed: false,
+      };
+      itemsArray.push(newTodo);
```

Comment:
- [ ] You are yet to implement the change requested by the previous reviewer. From the requirements, when a task is added it should have these properties: `description`, `index`, and `completed`, but from the image below you only have `description` and `completed`. Kindly include the missing `index` property.

  Also note that when a task is added, its `index` should be set to the length of the array. So for example, if I add the first task, its `index` should be `1`, and when I add the second task, its `index` should be `2`, and so on.

  ![jff](https://user-images.githubusercontent.com/36542986/194264954-d76daf25-67da-46b0-85ff-e9a2deb309c3.jpg)